style.css

```diff
@@ -1,25 +1,34 @@
 body {
   display: grid;
-  height: 5433.5px;
+  height: 6433.5px;
   justify-content: center;
   width: 375px;
   align-items: flex-start;
-  margin-left: 40%;
   background: #e5e5e5;
+  max-width: 100%;
```

Comment:
- [ ] As requested by the previous CR, please 🙏🏽 make your portfolio responsive to avoid the horizontal scroll bar for a better UI.

> Your project has a horizontal overflow which shows your project is not responsive so please make sure you adjust the style of your alignment of elements to remove the horizontal overflow.

Reply:
Hello, @franclobo  thanks for the helpful suggestion. Could you please guide me on how to remove the Horizontal overflow? I have tried different ways But I don't seem to succeed with it.  I will greatly appreciate your guidance. ( I have tried overflow: hidden) but that doesn't seem to solve the issue.